projects/components/src/header/page/page-header.component.ts

```diff
@@ -35,14 +35,7 @@ import { NavigableTab } from '../../tabs/navigable/navigable-tab';
           <ng-container *ngIf="this.contentAlignment === '${PageHeaderContentAlignment.Row}'">
             <ng-container *ngTemplateOutlet="this.projectedContentTemplate"></ng-container>
           </ng-container>
-          <ht-page-time-range
-            class="time-range"
-            *htIfFeature="'${ApplicationFeature.NavigationRedesign}' | htFeature; else globalTimeRangeTemplate"
-          ></ht-page-time-range>
-
-          <ng-template #globalTimeRangeTemplate>
-            <ht-time-range></ht-time-range>
-          </ng-template>
+          <ht-page-time-range class="time-range"></ht-page-time-range>
```

Comment:
this should be flagged by the time range flag

Reply:
As requested by Anand, there are two code paths in page-header, the one with time-range is behind the feature flag

Reply:
Got it. I just saw the most recent diff and the pagetimerange feature flag wasn't in it, just saw the FF removed. I see it now when unfolding.